test/e2e_tests/init.js

```diff
@@ -1917,7 +1924,7 @@ describe('e2e tests for init', function() {
     });
   });
 
-  it('make sure we do not send analytics data if allowAnalytics is set to false', async function() {
+  it('make sure we send analytics data by default', async function() {
```

Comment:
Keep this test like before only. This will be to test if we set the `answers.allowAnonymousMetrics` to `false` in future, the metrics won't be sent.

The above test already tests that the anonymous metrics are collected by default, because if `answers.allowAnonymousMetrics` is modified anywhere outside the `askQuestions` method and set to `false`, the above test would fail.

Reply:
I got your point now, updates pushed.